tests/contenttypes_tests/test_models.py

```diff
@@ -259,6 +259,22 @@ def test_missing_model(self):
         ct_fetched = ContentType.objects.get_for_id(ct.pk)
         self.assertIsNone(ct_fetched.model_class())
 
+    @mock.patch('django.apps.apps.get_model', return_value=Site)
+    def test_shortcut_view_with_empty_site_fk(self, get_model):
+        """
+        Check that the shortcut view catch an AttributeError raised when the
```

Comment:
State the expected behavior: "The shortcut view works if a model's ForeignKey to site is None." (ticket reference not needed since the issue is straightforward).

Also, this test goes in `tests.py` along with the rest of the view tests, not in the model tests.